datafusion/substrait/tests/cases/tpch.rs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//! tests contains in <https://github.com/substrait-io/consumer-testing/tree/main/substrait_consumer/tests/integration/queries/tpch_substrait_plans>
```

Comment:
This is very cool -- thank you . I think the context of this PR may be lost after merge so some more documentation might help

Something like

```suggestion
//! TPCH `substrait_consumer` tests
//!
//! This module tests that substrait plans as json encoded protobuf can be 
//! correctly read as DataFusion plans. 
//! 
//! The input data comes from  <https://github.com/substrait-io/consumer-testing/tree/main/substrait_consumer/tests/integration/queries/tpch_substrait_plans>
```

Reply:
> //! TPCH `substrait_consumer` tests
> //!
> //! This module tests that substrait plans as json encoded protobuf can be 
> //! correctly read as DataFusion plans. 
> //! 
> //! The input data comes from  <https://github.com/substrait-io/consumer-testing/tree/main/substrait_consumer/tests/integration/queries/tpch_substrait_plans>

Got it